.github/workflows/build-dist.yml

```
@@ -0,0 +1,18 @@
+name: Ensure Dist Build
+on:
+  push:
```

(e.g. so it won't apply on dependabot branches)

```suggestion
  push:
    branches:
      - main
```